### PR TITLE
Fix accompanist deprecations

### PR DESCRIPTION
**Pull request by twizmwazin**
~I have not fully tested this yet~

### CODE REVIEW

File: app/src/main/java/com/jerboa/ui/components/inbox/InboxActivity.kt [hunk 1]
```diff
@@ -143,7 +144,7 @@ enum class InboxTab {
     Messages,
 }
 
-@OptIn(ExperimentalPagerApi::class)
+@OptIn(ExperimentalFoundationApi::class, ExperimentalMaterialApi::class)
```
Comment (dessalines) [inline app/src/main/java/com/jerboa/ui/components/inbox/InboxActivity.kt:147]: Double :) . Android studio does this to me all the time too.

**Review by dessalines — COMMENTED** [review]
I'll review once you're satisfied w/ it. I'll mark as draft until then.

File: app/src/main/java/com/jerboa/ui/components/inbox/InboxActivity.kt (hunk 1 above)
Comment (twizmwazin) [reply app/src/main/java/com/jerboa/ui/components/inbox/InboxActivity.kt:147]: These are separate, removing either causes a build failure. Is there an opt-in that is a superset of these two?

Comment (dessalines) [reply app/src/main/java/com/jerboa/ui/components/inbox/InboxActivity.kt:147]: Ohhh my bad, I thought they were the same string at first glance.

File: app/build.gradle [hunk 2]
```diff
@@ -75,7 +75,6 @@ dependencies {
     // Accompanist
     implementation 'com.google.accompanist:accompanist-pager:0.30.1'
     implementation 'com.google.accompanist:accompanist-pager-indicators:0.30.1'
-    implementation 'com.google.accompanist:accompanist-swiperefresh:0.30.1'
```
Comment (dessalines) [inline app/build.gradle:78]: yaaay no more warnings from gradle builds!

**Review by dessalines — COMMENTED** [review]
One thing I noticed that's missing, was those handy pull refresh loading indicators. This issue shows how to add them: https://stackoverflow.com/a/73953806/1655478

My `autogen_types` branch reworks the viewmodels and API to show API loading states easier, but that's a bit away. For now just use the old isRefreshing logic (its very hacky and imperfect, but it works).

**Comment by twizmwazin** [comment]
Alright indicators are back!

**Review by dessalines — APPROVED** [review]
LGTM. Glad we can finally get rid of those warnings.

**Pull request merged**